.flake8

```diff
@@ -1,12 +1,14 @@
 [flake8]
-exclude=
-    .tox
-    setuptools/_vendor,
-    pkg_resources/_vendor
+max-line-length = 88
+exclude =
+	setuptools/_vendor
+	pkg_resources/_vendor
 ignore =
-    # W503 violates spec https://github.com/PyCQA/pycodestyle/issues/513
-    W503
-    # W504 has issues https://github.com/OCA/maintainer-quality-tools/issues/545
-    W504
-    setuptools/site-patch.py F821
-    setuptools/py*compat.py F811
+	# W503 violates spec https://github.com/PyCQA/pycodestyle/issues/513
+	W503
+	# W504 has issues https://github.com/OCA/maintainer-quality-tools/issues/545
+	W504
+	# Black creates whitespace before colon
+	E203
+	setuptools/site-patch.py F821
+	setuptools/py*compat.py F811
```

Comment:
I'm pretty sure `ignore` doesn't work like this. You need to use `per-file-ignores` for this to make sense to flake8

Reply:
The docs certainly don't mention this usage: https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-ignore

Reply:
I think there was `per-file-ignores =` line at some point in the past above these lines and it probably got removed by accident.

Reply:
I don't recall, but it falls outside the scope of this effort. I welcome others to investigate and improve in a separate PR.

Reply:
Sure, it's an external effort. I just wanted to point out that these lines most likely don't have any effect at all.